charts/attribute-masking-service/templates/configmap-akka-redis.yaml

```diff
@@ -28,11 +28,14 @@ data:
       kafka:
         consumer.topics:
           input-topic.partitions: {{ .Values.replicaCount }}
+          {{ if .Values.global.topicPrefix }}input-topic.name: {{ .Values.global.topicPrefix }}rule {{ end }}
         producer.topics:
           output-topic.partitions: {{ .Values.replicaCount }}
+          {{ if .Values.global.topicPrefix }}output-topic.name: {{ .Values.global.topicPrefix }}masked-resource {{ end }}
           error-topic.partitions: {{ .Values.replicaCount }}
+          {{ if .Values.global.topicPrefix }}error-topic.name: {{ .Values.global.topicPrefix }}error {{ end }}
       discovery.config.services.kafka.endpoints:
-      - host: {{ include "palisade.kafka.fullname" . }}-headless
+      - host: {{ include "palisade.kafka.fullname" . }}-headless{{ .Values.global.serviceName }}
```

Comment:
Would prefer a different mechanism, maybe consider if we can not use the headless svc

Reply:
this doesn't seem right, let's have a talk about it when you're ready